src/main/kotlin/no/nav/syfo/clients/HttpClients.kt

```diff
@@ -72,53 +69,48 @@ class HttpClients(env: Environment, vaultSecrets: VaultSecrets) {
 
     private val httpClient = HttpClient(Apache, config)
 
-    private val azureAdV2Client = AzureAdV2Client(
+    internal val azureAdV2Client = AzureAdV2Client(
```

Comment:
Hva er grunnen til at private er byttet ut med internal? Har ikke brukt internal selv og er usikker på hva den gjør :)

Reply:
Jeg samlet initialisering av services i i Bootstrap, mens klientene forble igjen i HttpClients. Det betyr at variabler som azureAdV2Client som tidligere kunne være privat for klassen nå må være tilgjengelig utenfor.

Internal betyr såvidt jeg har skjønt omtrent det samme som "package private" i java, men jeg er ikke bombesikker på forskjellen her. Jeg følte det var litt mer privat enn public :P